go.mod

```diff
@@ -176,6 +179,8 @@ require (
 	github.com/flosch/pongo2 v0.0.0-20200913210552-0d938eb266f3 // indirect
 	github.com/gdamore/encoding v1.0.0 // indirect
 	github.com/gdamore/tcell/v2 v2.5.1 // indirect
+	github.com/go-delve/delve v1.23.1 // indirect
```

Comment:
We do need to think about the implications here. We now have a dependency on delve, which means we also take on the burden of all that comes with it. In the past we had a policy of requiring a sign-off from at least one staff. We should do the same here.

Reply:
This dependency isn't shipped with the prod binary. Those will be shipped only if debug tags is used (through 
`DEBUG_JUJU=1 make install`)

```sh
❯ go list ./... | grep delve # return nothing

❯ go list -tags=debug ./... | grep delve
internal/dlv/examples/init_debug.go:10:2: no required module provides package github.com/juju/juju/internal/dlv/config; to add it:
        go get github.com/juju/juju/internal/dlv/config
```

Reply:
My point still stands.

>  In the past we had a policy of requiring a sign-off from at least one staff. We should do the same here.